client/src/redux/actions/observations.js

```diff
@@ -60,9 +78,26 @@ export const deleteObservations = userSession => async dispatch => {
 };
 
 export const fetchObservations = userSession => async dispatch => {
-  const observations = await getObservations(userSession);
-  const JSONobservations = JSON.parse(observations);
-  if (JSONobservations) {
-    dispatch({ type: FETCH_OBSERVATIONS, observations: JSONobservations });
-  }
+  return userSession.getFile('tempUnit').then(async tempUnit => {
```

Comment:
```suggestion
  const tempUnit = await userSession.getFile('tempUnit');
```

The mixture of `then` and `await` can be confusing